src/world.cpp

```diff
@@ -40,7 +33,7 @@
 
 const terrain::Material*
 World::get_material(MaterialId material_id) const {
-    return &materials.at(material_id);
+    return &biome_.get_materials().at(material_id);
```

Comment:
careful about this as ptrs are invalidated if a collection is resized

Reply:
I made materials constant